inspector/Readme.md

```diff
@@ -0,0 +1,58 @@
+# SumoLogic Lambda Function for Amazon Inspector
+
+This function receives the records published to a SNS Topic by Amazon Inspector.It looks up an Inspector object based on its arn and type and then adds extra context to the final messages which are compressed and send to Sumo Logic HTTP source endpoint.
+
+## Lambda Setup
+
```

Comment:
You can point to the dochub page: 
https://help.sumologic.com/Send-Data/Applications-and-Other-Data-Sources/Amazon-Inspector-App/01-Collect-Data-for-Amazon-Inspector